content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -16,79 +16,96 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # AWS Cluster Autoscaler
 
-Palette supports Autoscaling for our **EKS clusters** to maximise the benefits of the AWS Cloud utilization. Palette Auto Scaling ensures that instances are launched propotional to the workload on the EKS cluster. The resources are scaled up and down based on the changing workload on the cluster by continously monitoring the applications and automatically adjusts the capacity to maintain steady, predictable performance at the lowest possible cost.
+Palette supports autoscaling for AWS EKS clusters, ensuring cluster nodes launch according to the workload. 
 
+The AWS cluster autoscaler dynamically scales resources up or down based on the changing cluster workload. It monitors the workload and provisions or shuts down cluster nodes to maintain consistent cluster utilization. It resizes the Kubernetes cluster under the following conditions:
 
-# Prerequisite
+* Insufficient resources leading to multiple pod failures in the cluster. The AWS cluster autoscaler redistributes these pods to different nodes in such cases.
+
+* Underutilized cluster nodes for a specific period. In this scenario, the AWS cluster autoscaler migrates the pods from underutilized nodes to other available nodes.
 
-* Kubernetes version 1.19.x and above.
-* Full Cluster Autoscaler Policy for the service account.
-* Update the Kubernetes Pack node group `managedMachinePool` with the **ARN** of the autoscaler policy created for the service account.
-
-
-## Full Cluster Autoscaler Features Policy (Recommended)
-
-```json
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:DescribeAutoScalingGroups",
-        "autoscaling:DescribeAutoScalingInstances",
-        "autoscaling:DescribeLaunchConfigurations",
-        "autoscaling:DescribeTags",
-        "ec2:DescribeInstanceTypes",
-        "ec2:DescribeLaunchTemplateVersions"
-      ],
-      "Resource": ["*"]
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "autoscaling:SetDesiredCapacity",
-        "autoscaling:TerminateInstanceInAutoScalingGroup",
-        "ec2:DescribeInstanceTypes",
-        "eks:DescribeNodegroup"
-      ],
-      "Resource": ["*"]
-    }
-  ]
-}
-```
-<InfoBox>
-
-To deploy AWS Autoscaler, update the Kubernetes pack `managedMachinePool` node group with the **ARN** of the autoscaler policy created for the service account.
-
-```yaml
-managedMachinePool:
-  #roleName: {{ name of the self-managed role | format "${string}" }}
-
-  ## A list of additional policies to attach to the node group role
-  roleAdditionalPolicies:
-  - "arn:aws:iam::012345678910:policy/autoscalingpolicy"
-```
-</InfoBox>
-
-
-## Versions Supported
+# Versions Supported
 
 <Tabs>
 
-<Tabs.TabPane tab="1.22.x" key="1.22.x">
+<Tabs.TabPane tab="1.24+" key="1.24.x">
+
+**1.24+**

```

Comment:
The versions supported are `1.22.2` and `1.26.3` . 1.24 is the k8s version that supports these packs